pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -228,6 +229,22 @@ public static long fromEpochDaysBucket(long days, long bucket) {
     return TimeUnit.DAYS.toMillis(days * bucket);
   }
 
+  /**
+   * Converts epoch millis to Timestamp
+   */
+  @ScalarFunction
+  public static Timestamp toTimestamp(long millis) {
+    return new Timestamp(millis);
+  }
+
+  /**
+   * Converts Timestamp to epoch millis
+   */
+  @ScalarFunction
+  public static long fromTimestamp(Timestamp timestamp) {
```

Comment:
(nit) consider naming it fromTimestampToMillisSinceEpoch or something like that ?

Reply:
Also, what about adding other utility functions like adding/subtracting two timestamps. I think users can probably use ADD/SUB non-scalar transform function on TIMESTAMP type but probably this is a better place

Reply:
This follows the naming convention of other functions (e.g. `fromDateTime`, `fromEpochDays` etc.)
We can add more util functions in the following PRs as needed. That should be very straight forward